### DISCUSSION TITLE
cannot switch between Chrome OS and Ubuntu

### DISCUSSION THREAD

**Discussion by bradfc** [dnschneid/crouton#4637] [Q&A]
I'm running a new Lenovo Flex 5 with Intel i3-1115G4 and have installed Ubuntu 20.04+MATE.  Per the guidance for Intel processors, I've tried CTRL-ALT-BACK and CTRL-ALT-FORWARD (tried using SHIFT as well) within an Ubuntu session, and it does not toggle back to Chrome OS.  If I'm in a browser, those keystrokes simply move me forward and backward through the windows in the browser session.  If I'm in another application or simply the open space on the screen, nothing happens.

Is this the right procedure, or is it possibly related to Ubuntu 20.04 or MATE?

I would not normally switch to Chrome OS at all, but it looks like I have to in order to set up wi-fi in a new location.

**Comment by DennisLfromGA** [comment]
@bradfc,

You didn't show us the output of `sudo edit-chroot -all` so we don't know what xmethod(s) you installed but if you installed and are using the `xorg` xmethod then you can flip through your running chroot desktops and Chromium OS by hitting
<kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>Shift</kbd>+<kbd>Back</kbd> and <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>Shift</kbd>+<kbd>Forward</kbd>. 

You can also use `croutoncycle` in a crouton chroot shell to switch back to CrOS.
See the `croutoncycle` usage below:

```
croutoncycle -h
croutoncycle next|prev|cros|list|#
Cycles through running graphical chroots.
    next: switch to the next display
    prev: switch to the previous display
    cros: switch directly to Chromium OS
    list: list all of the active displays and the associated chroot name
    #: switch to the nth item in the list, zero-indexed
    :#: switch directly to the chroot owning the specified display number
```

You can use the '[crouton integration](https://chrome.google.com/webstore/detail/crouton-integration/gcpneefbbnfalgjniomfjknbcgkbijom)' extension in CrOS to switch *into* your chroot(s)

*Hope this helps,*
-**DennisLfromGA**

**Reply by bradfc** [reply]
I'm afraid I'm new to this...

Here is the output:

name: focal
encrypted: no
crouton: version 1-20210907212903~master:9b281df5
release: focal
architecture: amd64
xmethod: xorg
targets: xfce,unity
host: version 14150.87.0 (Official Build) stable-channel volteer 
kernel: Linux Curie 5.4.139-16311-g330b3df0010b #1 SMP PREEMPT Tue Nov 9 13:45:00 PST 2021 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
name: xenial
encrypted: no
crouton: version 1-20210907212903~master:9b281df5
release: xenial
architecture: amd64
xmethod: xorg
targets: unity
host: version 14150.87.0 (Official Build) stable-channel volteer 
kernel: Linux Curie 5.4.139-16311-g330b3df0010b #1 SMP PREEMPT Tue Nov 9 13:45:00 PST 2021 x86_64 x86_64 x86_64 GNU/Linux
freon: yes

It's actually running MATE (I got it running through some of your comments on Issue 4079), though it's not listed as a 'target' - maybe that is the problem.  But xorg is listed, though the requisite keystrokes have no effect.

**Comment by DennisLfromGA** [comment]
@bradfc,

Okay, you *are* using the `xorg` xmethod so the key-combos I listed above *should* work.
If they don't then use the `croutoncycle` command shown above in a chroot shell session.

*Hope this helps,*
-**DennisLfromGA**
